components/user/UserSocials.js

```diff
@@ -2,31 +2,17 @@ import getIcon from "../Icon";
 
 function UserSocial({ BASE_URL, username, social }) {
   const DisplayIcon = getIcon(social.icon);
-  const clickLink = async () => {
-    try {
-      const res = await fetch(
-        `${BASE_URL}/api/statistics/${username}/${encodeURIComponent(
-          social.url
-        )}`,
-        { method: "PUT" }
-      );
-      const data = await res.json();
-      setClicks(data.clicks);
-    } catch (e) {
-      // TODO: link not found
-      console.log("ERROR link not found ", e);
-    }
-  };
 
   return (
     <a
-      href={social.url}
-      onClick={clickLink}
+      href={`${BASE_URL}/api/handler/links?username=${username}&url=${encodeURIComponent(
```

Comment:
They're also present here.

Reply:
I don't know why there should be quotes. I've ran some manual tests and the link works 👍